src/cmd/script.rs

```diff
@@ -1,19 +1,22 @@
+use super::script_args::ScriptFunctionArguments;
+use super::script_transaction::ScriptTransaction;
+use crate::run_context::RunContext;
 use anyhow::{Error, Result};
 use clap::Parser;
 use move_binary_format::file_format::CompiledScript;
 use std::fs;
 use std::path::PathBuf;
 
-use super::script_transaction::ScriptTransaction;
-use crate::run_context::RunContext;
```

Comment:
Why did you move the crate-internal imports to the top? I'm asking, because in the past, I usually did the opposite 🙃

Reply:
That's not my work. That was done by the all-mighty:
`cargo +nightly fmt --all`